packages/@guardian/source-react-components-development-kitchen/src/star-rating/StarRating.tsx

```diff
@@ -32,7 +31,7 @@ const determineSize = (size: Size) => {
 	}
 };
 
-export interface StarRatingProps extends Props {
+export interface StarRatingProps {
```

Comment:
Is it fine not to use the standard `Props` here? Mainly curious, unsure of the rules Source recommends 🤔

Reply:
It says `Props` is deprecated, hence why I’ve removed it.